client/modules/Airowl/Airowl.js

```diff
@@ -102,9 +105,10 @@ export default class Airowl extends Component{
                             <div className="text-center red margin-bottom-10p">
                               <i className="fa fa-inr" style={{marginRight: "5px"}}></i> 5,000/-
                             </div>
-                            <a href="https://www.instamojo.com/AirOwl/airowl-wi/" rel="im-checkout" data-behaviour="remote" data-style="flat" data-text="Buy AirOwl Wi" data-token="4aaf75271bbcbaa4e9d9bc35a38433d1">
-
-                            </a>
+                             {/*<a href="https://www.instamojo.com/AirOwl/airowl-wi/" rel="im-checkout" data-behaviour="remote"
+                            data-style="flat" data-text="Buy AirOwl Wi" data-token="4aaf75271bbcbaa4e9d9bc35a38433d1">
+                            </a> */}
+                              <Button style={{color:"#fff", background:"#C30F1F", borderRadius:"1.5em", padding:"0.6em", width: "250px", borderColor: "#C30F1F", margin:'20px' }} disabled>B̶u̶y̶ ̶A̶i̶r̶O̶w̶l̶ ̶W̶i̶  (Sold Out)</Button>
```

Comment:
How about using `<strike>` for using strikethough. 

Also, style is getting duplicated, possible if to reuse something.